### ISSUE TITLE
Allow setting APP_KEY from env variables

### ISSUE THREAD

**Issue by Arno500** [linuxserver/docker-bookstack#57]
## Expected Behavior

I want to be able to seed an APP_KEY env variable (stored in a secret in Kubernetes for example), and the container should take it instead of regenerating one and forcefully putting it in the .env

## Current Behavior

Currently if I destroy the container (only keeping the uploads, that's what should be immutable), it will recreate an APP_KEY again, thus invalidating the data in database. The DB_PASS/USER can be passed again, so this is not a problem for it.

## Steps to Reproduce

1. Destroy the container (and the volume if configured on /config), try to replug it to the DB.

## Environment
**OS:** Ubuntu
**CPU architecture:** x86_64
**How docker service was installed:** On Kubernetes with a Helm chart

**Comment by homerr** [comment]
Had a look into this one, aware it is a little old but I remember having this dilemma when I originally set up the config scripts and the bit below should handle the issue you have detailed;

`# Create API key if needed
if [ ! -f "/config/BOOKSTACK_APP_KEY.txt" ];
	then
	echo "Generating BookStack app key for first run"
	key=$(php /var/www/html/artisan key:generate --show)
	echo $key > /config/BOOKSTACK_APP_KEY.txt
	echo "App Key set to $key you can modify the file to update /config/BOOKSTACK_APP_KEY.txt"
elif [ -f "/config/BOOKSTACK_APP_KEY.txt" ];
	then
	echo "App Key found - setting variable for seds"
	key=$(cat /config/BOOKSTACK_APP_KEY.txt)
fi`

Basically providing the directory is not lost when you destroy the container, providing it is linked to the new container it should detect the app key and recycle it. 

If this isn't happening, might need some more looking in to but can't see how it wouldn't work providing the original files are in place.

**Comment by homerr** [comment]
Going to close this, issue was way old (sorry!) and I think everything works as intended. Let us know if not...
